tests/test_day_at_the_supermarket.py

```diff
@@ -5,23 +5,23 @@
 from codecademy.day_at_the_supermarket import compute_bill
 
 
-@pytest.mark.parametrize("test_compute_bill_input,test_compute_bill_expected", [
+@pytest.mark.parametrize("input,expected", [
     (["banana"], 4),
     (["banana", "orange", "pear"], 8.5),
     (["banana", "orange", "pear", "apple"], 8.5),
     (["banana", "orange", "orange", "pear", "apple"], 10),
     (["banana", "banana", "orange", "pear", "apple"], 12.5)
 ])
-def test_compute_bill(test_compute_bill_input, test_compute_bill_expected):
-    assert compute_bill(test_compute_bill_input) == test_compute_bill_expected
+def test_compute_bill(input, expected):
+    assert compute_bill(input) == expected
 
 
-@pytest.mark.parametrize("test_compute_bil_negative_input", [[], [0, 1, 3, False, None], ["orange", "pear"],
-                                                             ["orange", "pear", "potato"],
+@pytest.mark.parametrize("input", [[], [0, 1, 3, False, None], ["orange", "pear"], ["orange", "pear", "potato"],
                                                              ["banana", "banana", "orange", "pear", "apple"],
                                                              0, False, True, None])
-def test_compute_bill_negative(test_compute_bil_negative_input):
+def test_compute_bill_negative(input):
     try:
-        compute_bill(test_compute_bil_negative_input)
+        compute_bill(input)
+        assert str(False) + " error enter type"
```

Comment:
@09041985 
in your code you cast False boolean value into a string and then append " error enter type", so this will result in
```python
assert "False error enter type"
```
which will always pass... but you need to raise an error because test execution should not reach this line because we expect exception.

So this should be just `assert False, "Your custom error message` - this line would raise assertion error.